ssl/quic/quic_port.c

```diff
@@ -765,19 +764,20 @@ static int parse_validation_token(QUIC_VALIDATION_TOKEN *token,
         || !PACKET_copy_bytes(&pkt, (unsigned char *)&token->timestamp,
                               sizeof(token->timestamp))
         || (token->is_retry
-            && (!PACKET_copy_bytes(&pkt, &token->odcid.id_len,
-                sizeof(token->odcid.id_len))
-                || token->odcid.id_len > QUIC_MAX_CONN_ID_LEN
-                || !PACKET_copy_bytes(&pkt, (unsigned char *)&token->odcid.id,
+            && (!PACKET_get_length_prefixed_1(&pkt, &subpkt)
+                || (token->odcid.id_len = PACKET_remaining(&subpkt))
+                > QUIC_MAX_CONN_ID_LEN
```

Comment:
Style nit: Line up the `>` with the `(` on the previous line

Reply:
That's what I originally tried, but the style checker forced me to align it as it is currently.

Reply:
The sytle checker is wrong IMO. Since we're waiving anyway, please do it.